authors/apps/articles/views.py

```diff
@@ -58,6 +82,12 @@ def post(self, request):
         serializer = self.serializer_class(data=article)
         serializer.is_valid(raise_exception=True)
         serializer.save()
```

Comment:
@meshack-mbuvi here you can save the serializer/model instance on save if you need to use it after a save is performed. For instance:
`saved_article = serializer.save()`
You can the user the `saved_article` object in the `create_tag` method in line 90 of articles/views.py.
i.e `create_tag(tags, saved_article)`
This approach means that we don't have to query the database for the saved article as you have done in line 89

Reply:
This approach can also be used in the definition of the `create_tag` method where you have done `new_tag = ArticleTags.objects.get(tag=tag.strip())`

Reply:
Thanks very much. I have learned something from this.

Reply:
Check the new changes and if you see anything I can improve, please let me know; it really helps a lot.

Reply:
@meshack-mbuvi great work resolving the issues. This PR now looks good to me.

Reply:
This is very helpful.